src/mono/wasm/debugger/DebuggerTestSuite/Tests.cs

```diff
@@ -826,6 +826,33 @@ public async Task GetSourceUsingSourceLink()
             Assert.True(source.IsOk);
         }
 
+        [Fact]
+        public async Task GetObjectValueWithInheritance()
```

Comment:
We have some tests for inherited members in https://github.com/dotnet/runtime/blob/main/src/mono/wasm/debugger/DebuggerTestSuite/GetPropertiesTests.cs . Can you add any missed cases there? or the new tests in that file?

Reply:
done.